app/services/search/supergroups.rb

```diff
@@ -0,0 +1,29 @@
+module Search
+  class Supergroups
+    SUPERGROUP_TYPES = %w(
```

Comment:
Maybe we should use the document_types gem here:
```
    def supergroup_types
      @supergroup_types ||= GovukDocumentTypes::SUPERGROUPS.dig('content_purpose_supergroup', 'items').map { |item| item['id'] }
    end
```

what do you think?

Reply:
In principle that's a good idea, but I worry if we add a new supergroup then this will automatically start appearing on org pages.

Reply:
@bilbof It should, but only if there are any documents of those types tagged to the organisation. The bonus is we have less places to maintain the list of supergroups.

Reply:
Awesome, lets change it :)